metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -80,49 +80,7 @@ class DataflowRecipe:
     @property
     def join_targets(self) -> List[JoinDescription]:
         """Joins to be made to source node."""
-        join_targets = []
-        for join_recipe in self.join_linkable_instances_recipes:
```

Comment:
Moved this logic to `JoinLinkableInstancesRecipe`, which seemed cleaner & more appropriate.